addons/mail/tests/discuss/test_discuss_channel.py

```diff
@@ -540,6 +540,7 @@ def test_mail_message_starred_group(self):
         message in a group should be considered only if:
             - It's our message
             - OR we have access to the channel
+            - OR you are a member of the channel
```

Comment:
This is included in the `OR we have access to the channel` condition just above.